### PR TITLE
rename and update some variables for the test-eng team rename

### PR DESCRIPTION
**Pull request by stuartphilp**
a quick once over, probably missed some things :P

@davehunt @m8ttyB anyone else :) r?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,9 +57,9 @@
           <li id="nav-jenkins"><a href="https://webqa-ci.mozilla.com/">Jenkins</a></li>
         </ul>
         <ul class="nav nav-sidebar">
-          <li><a href="https://quality.mozilla.org/teams/web-qa/">About Web QA</a></li>
-          <li><a href="https://wiki.mozilla.org/QA/Execution/Web_Testing">Wiki</a></li>
-          <li><a href="https://oneanddone.mozilla.org/tasks/available/?search=&team=6">Contribute</a></li>
+          <li><a href="https://quality.mozilla.org/teams/test-engineering/">About Firefox Test Engineering</a></li>
```
Comment (m8ttyB) [inline index.html:60]: Is this a 'coming soon' page?

**Comment by m8ttyB** [comment]
r+

**Comment by rbillings** [comment]
R+ with the same comment as @m8ttyB. Also, it would be nice if there was a One and Done team assignation as that gives more of a focus to the contributor- but that's a fix outside of this PR.

File: index.html (hunk 1 above)
Comment (stuartphilp) [reply index.html:60]: yeah asked john to take a look at updating qmo (with that specific url)

**Comment by stuartphilp** [comment]
yup! we need to look at oneanddone team tasks and see what makes sense. can follow up :)

**Pull request merged**